test/test_models.py

```diff
@@ -69,16 +91,255 @@ def check_script(self, model, name):
+    def test_squeezenet1_0(self):
+        self._test_classification_model(models.squeezenet1_0)
+
+    def test_squeezenet1_1(self):
+        self._test_classification_model(models.squeezenet1_1)
+
+
+# TODO add test for width_mult arg to factory method
+class MobilenetTester(ClassificationModelTester):
+    def test_mobilenet_v2(self):
+        self._test_classification_model(models.mobilenet_v2)
+
+    def test_mobilenetv2_residual_setting(self):
+        self._test_classification_model(models.mobilenet_v2, inverted_residual_setting=[[1, 16, 1, 1], [6, 24, 2, 2]])
+
+
+# TODO add test for aux_logits arg to factory method
+# TODO add test for transform_input arg to factory method
+class GooglenetTester(ClassificationModelTester):
+    def test_googlenet(self):
+        self._test_classification_model(models.googlenet)
+
+
+class VGGNetTester(ClassificationModelTester):
+    def test_vgg11(self):
+        self._test_classification_model(models.vgg11)
+
+    def test_vgg11_bn(self):
+        self._test_classification_model(models.vgg11_bn)
+
+    def test_vgg13(self):
+        self._test_classification_model(models.vgg13)
+
+    def test_vgg13_bn(self):
+        self._test_classification_model(models.vgg13_bn)
+
+    def test_vgg16(self):
+        self._test_classification_model(models.vgg16)
+
+    def test_vgg16_bn(self):
+        self._test_classification_model(models.vgg16_bn)
+
+    def test_vgg19(self):
+        self._test_classification_model(models.vgg19)
+
+    def test_vgg19_bn(self):
+        self._test_classification_model(models.vgg19_bn)
+
+
+# TODO add test for dropout arg to factory method
+class MNASNetTester(ClassificationModelTester):
+    def test_mnasnet0_5(self):
+        self._test_classification_model(models.mnasnet0_5)
+
+    def test_mnasnet0_75(self):
+        self._test_classification_model(models.mnasnet0_75)
+
+    def test_mnasnet1_0(self):
+        self._test_classification_model(models.mnasnet1_0)
+
+    def test_mnasnet1_3(self):
+        self._test_classification_model(models.mnasnet1_3)
+
+
+# TODO add test for bn_size arg to factory method
+# TODO add test for drop_rate arg to factory method
+class DensenetTester(ClassificationModelTester):
+    def _test_densenet_plus_mem_eff(self, model_callable):
+        model, test_input, test_output = self._test_classification_model(model_callable)
+
+        # above, we perform the standard correctness test against the test fixture, and capture key test params
+        # below, we check that memory efficient/time inefficient DenseNet implementation behaves like the "standard" one
+        me_model = self._get_test_model(model_callable, num_classes=STANDARD_NUM_CLASSES, memory_efficient=True)
+        me_model.load_state_dict(model.state_dict())  # xfer weights over
+        me_output = self._infer_for_test_with(me_model, test_input)
+        test_output.squeeze(0)
+        me_output.squeeze(0)
+        self.assertTrue((test_output - me_output).abs().max() < EPSILON)
```

Comment:
Could we just use `assertNestedTensorObjectsEqual` or use the `assertEqual` that is in @driazati PR here https://github.com/pytorch/vision/pull/1430 ? both of these methods already have a tolerance arg builtin to them. 

Also since EPSILON is only used once i think it would be better to inline the usage here.

Reply:
I'm happy to switch over to using a more idiomatic assertion call. I'll look into the ones you flagged.

I'm less jazzed about taking away the label, even if it's only used once. Numbers have no semantic content. Not a huge deal in this case, but I prefer a name like `EPSILON` or `TOLERANCE` to a bare constant every time.

Reply:
Cool - named arguments serve the semantic purpose. I'll tweak this.